lib/pure/async.nim

```diff
@@ -0,0 +1,6 @@
+when defined(js):
+    import asyncjs
+    export asyncjs
+else:
+    import asyncmacro, asyncfutures
```

Comment:
I think ``asyncjs`` can use a lot of ``asyncmacro``'s functionality so I don't think this is correct.

Reply:
You may be right, but in any case this `async` module presents a single async/await/future interface, which should not change even if `asyncjs` will reuse `asyncmacro` in some way.